src/lib.rs

```diff
@@ -456,72 +531,69 @@ ImplDekuTraits!(isize);
 ImplDekuTraits!(f32);
 ImplDekuTraits!(f64);
 
+/*
+FIXME: Currently those implementation will expose it's implementation detail.
```

Comment:
Confused what you mean by this comment

Reply:
I will remove it. It means a user can know it will read a u32 through the bound `u32: DekuRead<Ctx>`. It's find because everyone knows Ipv4 is u32 and Ipv6 is u128

Reply:
It could be done in https://github.com/sharksforarms/deku/pull/61/files/5e7deecaeda49f31229b2d691aa0367791a3485d#diff-3227a35819c9da6bf1bf643fad7e99a8R186